modules/todoitems.js

```diff
@@ -0,0 +1,166 @@
+import select2 from '../img/select.png';
+import delete2 from '../img/delete.png';
+import edit from '../img/edit.png';
+import save from '../img/save.png';
+
+export default class toDoItems {
+  constructor(text, completed, index) {
+    this.text = text;
+    this.completed = completed;
+    this.index = index;
+  }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222570139-f5a35bd6-0225-4c05-8c9c-4e6c5444db00.png)

- [ ] It would be nice if you make your Getting Started Section in your readme file to make your readme file descriptive and professional 😊

## Getting Started

### Prerequisite

```
  Visual Studio Code
  Node.js
  Git
```

### Setup
```
  cd my-folder
  git clone https://github.com/fuadmoin/TODOLIST.git
  cd TODOLIST
```
### Install
```
  npm install
```
### Usage

```
  npm run build
  npm start
```
### Run tests
```
  npm test
```